### DISCUSSION TITLE
Paddleocr在训练文本识别模型时，acc精度一直为0.0000？怎么回事？是数据集路径的问题吗？还是怎么回事

### DISCUSSION THREAD

**Discussion by Liusdaooo** [PaddlePaddle/PaddleOCR#13798] [Q&A]
如图所示，在使用paddleocr 文本识别 ch_PP-OCRv3_rec.yml 识别字符时，参数如下
![6666666](https://github.com/user-attachments/assets/e4195d39-b551-4ad9-b285-a2d041341675)
![11111](https://github.com/user-attachments/assets/cfebe8dc-25ec-4e14-b129-1f82400f29bf)
![222222](https://github.com/user-attachments/assets/f0318251-f4e5-4b16-b434-a279dae0ba5d)
![555555](https://github.com/user-attachments/assets/0df59cfd-b2d3-42a5-bd8e-8c02835be943)
![3333333](https://github.com/user-attachments/assets/0d2ad36d-d047-411b-9c91-45b98484be5a)
![4444444](https://github.com/user-attachments/assets/445e7ff3-a9cb-4cb0-b9e7-4327130dc490)
文本识别训练集 如下：
![8888888888888](https://github.com/user-attachments/assets/aed196d5-7e28-485c-9452-c33956c19d28)
运行效果如下：
![8888888888888](https://github.com/user-attachments/assets/8351a4cf-99f3-41aa-8c74-901d109a84c3)
训练精度一直为0.000000

**Comment by echochoc** [comment]
请问找到原因了吗？我使用v4也是这个问题

**Comment by cuicheng01** [comment]
数据量是多少呢
